app/models/gsa18f/procurement.rb

```diff
@@ -122,5 +122,9 @@ def self.user_with_role(role_name)
 
       users.first
     end
+
+    def self.expense_type_options
```

Comment:
thoughts on defining this in `client_data_mixin` rather than here?  if there is an option to define a method as an empty array, maybe that should be the default rather than requiring all client models to define this?